modules/persistent-storage-local-discovery.adoc

```diff
@@ -21,7 +21,7 @@ Use the following procedure to automatically discover local devices, and to auto
 
 [WARNING]
 ====
-Use the `LocalVolumeSet` object with caution. When you automatically provision persistent volumes (PVs) from local disks, the local PVs might claim all devices that match. If you are using a `LocalVolumeSet` object, make sure the Local Storage Operator is the only entity managing local devices on the node.
+Use the `LocalVolumeSet` object with caution. When you automatically provision persistent volumes (PVs) from local disks, the local PVs might claim all devices that match. If you are using a `LocalVolumeSet` object, make sure the Local Storage Operator is the only entity managing local devices on the node. Ceating multiple instances of `LocalVolumeSet` that target a node more than once is not supported.
```

Comment:
```suggestion
Use the `LocalVolumeSet` object with caution. When you automatically provision persistent volumes (PVs) from local disks, the local PVs might claim all devices that match. If you are using a `LocalVolumeSet` object, make sure the Local Storage Operator is the only entity managing local devices on the node. Creating multiple instances of a `LocalVolumeSet` that target a node more than once is not supported.
```

Reply:
I also wonder about rewording this sentence:
There is no support for creating multiple instances of a `LocalVolumeSet` that target a node more than once.